### DISCUSSION TITLE
Trying to get the example to run, java.lang.NoClassDefFoundError: ch/bailu/gtk/gtk/Widget

### DISCUSSION THREAD

**Discussion by wusspuss** [bailuk/java-gtk#33] [Q&A]
I add the library via maven/jitpack as instructed in README. It builds fine, but when running it errors out with: 
`/usr/lib/jvm/java-1.11.0-openjdk-amd64/bin/java -javaagent:/home/archie/Downloads/idea-IC-233.14015.106/lib/idea_rt.jar=34095:/home/archie/Downloads/idea-IC-233.14015.106/bin -Dfile.encoding=UTF-8 -classpath /home/archie/IdeaProjects/example-gtk/target/classes org.example.Main
Error: Unable to initialize main class org.example.Main
Caused by: java.lang.NoClassDefFoundError: ch/bailu/gtk/gtk/Widget` 
![Screenshot from 2024-01-28 00-55-42](https://github.com/bailuk/java-gtk/assets/98534643/4316b9da-5c7d-42e1-af50-498089834287)
![Screenshot from 2024-01-28 00-55-57](https://github.com/bailuk/java-gtk/assets/98534643/99279ec6-50c3-4b5b-a7c3-2f09ad337495)

**Comment by bailuk** [comment]
The project builds when building with maven? 
To me it looks like VS-Code, respectively the java-plugin your using does not load dependencies correctly. Reloading dependencies with that maven icon does not help?
